types/theme.ts

```diff
@@ -2,5 +2,7 @@
 
 export enum Theme {
   Light = 'light',
-  Dark = 'dark'
+  Dark = 'dark',
+  System = 'system',
+
 }
```

Comment:
There's an extra newline in this enum.
Also a trailing comma when there wasn't one before. Don't know if this really matters in TS.
```suggestion
  System = 'system',
}
```